src/main/resources/static/general.css

```diff
@@ -0,0 +1,40 @@
+.danger-color {
```

Comment:
Am I right that this file is only used for the "still checked in message"? Then pls give it a more descriptive name such as checked-in-alert.css or so.